Dockerfile

```diff
@@ -1,24 +1,30 @@
 FROM factual/docker-base-openjdk
```

Comment:
We might want to change the base image here since [this one](https://github.com/Factual/docker-base-openjdk) seems like it's unmaintained? You also install openjdk below so you can probably use [factual/docker-base](https://github.com/Factual/docker-base) instead since it's more up-to-date.